packages/mars-theme/src/components/index.js

```diff
@@ -41,6 +42,31 @@ const Theme = ({ state }) => {
           (data.isPostType && <Post />) ||
           (data.is404 && <Page404 />)}
       </Main>
+
+      <Iframe
```

Comment:
Same here, remember not to commit these changes.

Reply:
Sure